File: baekjoon/17608.cpp
#include <stdio.h>

int main(void)
{
    int n, sticks[100000],max,count;
    scanf("%d",&n);
    count = 0;
    max = 0;
    for (int i = n-1; i>=0; i--)
    {
        scanf("%d\n",&sticks[i]);
    }
    for (int i = 0; i<n; i++)
    {
        if (max < sticks[i])
        {
            max = sticks[i];
            count++;
        }
    }
    printf("%d\n",count);
}

File: baekjoon/17608.py
from sys import stdin
numOfSticks = int(stdin.readline().strip())
sticks = list(reversed([ int(stdin.readline().strip()) for _ in range(numOfSticks)]))
count = 1
nowMax = sticks[0]
for i in sticks:
    if i > nowMax:
        count+=1
        nowMax = i
print(count)
